pkg/geo/geo.go

```diff
@@ -41,6 +41,11 @@ type Geography struct {
 	SRID   geopb.SRID
 }
 
+// NewGeography constructs a new Geography with the given figure and SRID.
+func NewGeography(srid geopb.SRID, figure ...s2.Region) *Geography {
```

Comment:
wondering if we want to make this return a non-ptr version and call this `MakeGeography` instead....